### PR TITLE
Consider porting rddlist from sparkapi to sparklyr

### PR DESCRIPTION
**Pull request by javierluraschi**
Proposal to port `rddlist` to use sparklyr since sparkapi is no longer active. Neat to see this working locally! 👍

### CODE REVIEW

File: R/rddlist.R [hunk 1]
```diff
@@ -30,7 +30,7 @@ rddlist <- function(sc, X, cache = TRUE) {
     # An RDD of the serialized R parts This is class
     # org.apache.spark.api.java.JavaRDD
     RDD = invoke_static(sc, "org.apache.spark.api.r.RRDD", "createRDDFromArray",
-        sparkapi:::java_context(sc), serial_parts)
+        sparklyr:::java_context(sc), serial_parts)
```
Comment (clarkfitzg) [inline R/rddlist.R]: Could this `java_context` function be exported?

**Comment by clarkfitzg** [comment]
Let's just have 1 README file. It's fine if it has the `.Rmd` extension if that's easier for RStudio users.

File: derby.log [hunk 2]
```diff
@@ -0,0 +1,13 @@
+----------------------------------------------------------------
```
Comment (clarkfitzg) [inline derby.log]: No need to include log file. Maybe we could add `*.log` to `.gitignore`.

Comment (javierluraschi) [reply derby.log]: Ah, yes, removed.

File: R/rddlist.R (hunk 1 above)
Comment (javierluraschi) [reply R/rddlist.R]: Yes, for sure. Using exported prefix now.

**Comment by javierluraschi** [comment]
@clarkfitzg feel free to remove the `.Rmd` file; however, if you open this in RStudio Preview, it's really worth it since you would be able to run your README file as an R notebook and knit the output directly to the `.md` file, but up to you. I can pull it out if you preffer to keep this project agnostic.

**Comment by clarkfitzg** [comment]
Thanks @javierluraschi!

**Pull request merged**